### PR TITLE
Update source-editor.html

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/assets/source-editor.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (XhmikosR) [inline app/assets/source-editor.html]: Why is this even needed? This is Internet Explorer specific.

File: app/assets/source-editor.html [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+    <meta http-equiv="Content-Language" content="en">
```
Comment (XhmikosR) [inline app/assets/source-editor.html]: Pretty much obsolete. `<html lang="en">` is simpler.

**Comment by hzsweers** [comment]
@ghost can you respond to @XhmikosR's review comments?

**Comment by jonan** [comment]
The user that created this PR has deleted his account no point in leaving this open

**Pull request closed without merging**